buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -661,10 +661,10 @@ class BeamModulePlugin implements Plugin<Project> {
         proto_google_cloud_bigtable_admin_v2        : "com.google.api.grpc:proto-google-cloud-bigtable-admin-v2", // google_cloud_platform_libraries_bom sets version
         proto_google_cloud_bigtable_v2              : "com.google.api.grpc:proto-google-cloud-bigtable-v2", // google_cloud_platform_libraries_bom sets version
         proto_google_cloud_datacatalog_v1beta1      : "com.google.api.grpc:proto-google-cloud-datacatalog-v1beta1", // google_cloud_platform_libraries_bom sets version
-        proto_google_cloud_datastore_v1             : "com.google.api.grpc:proto-google-cloud-datastore-v1", // google_cloud_platform_libraries_bom sets version
+        proto_google_cloud_datastore_v1             : "com.google.api.grpc:proto-google-cloud-datastore-v1:0.93.10", // google_cloud_platform_libraries_bom sets version
```

Comment:
My memo: This is in-line with com.google.cloud.datastore:datastore-v1-proto-client:2.2.10

(no action required)

Reply:
@ansh0l For upcoming change by you, remove this version part and let google_cloud_platform_libraries_bom set the version.

Reply:
This change suggests changes in the version for `proto_google_cloud_datastore_v1`. Should we let the Datastore team make this change, and not us (Spanner team)?

Reply:
Can you (spanner team) take care of this?

Reply:
Sure, we'll take care of this.